dm/syncer/causality.go

```diff
@@ -137,3 +142,72 @@ func (c *causality) detectConflict(keys []string) bool {
 
 	return false
 }
+
+type versionedMap struct {
+	data            map[string]string
+	prevFlushJobSeq int64
```

Comment:
what about rename versionedMap itself? Since each `versionedMap` means a group of keys that committed after `prevFlushJobSeq`, i am thinking of a name like `DmlJobKeysGroup` or similar ones.

Reply:
lgtm